lib/mix/test/mix/tasks/test_test.exs

```diff
@@ -257,6 +257,55 @@ defmodule Mix.Tasks.TestTest do
     end)
   end
 
+  test "--include, --exclude, and --only" do
```

Comment:
Those tests are very very very expensive to run, that's why we don't run all possible combination. Instead we unit test the `ex_unit_opts` function and make sure we are passing the proper options to ExUnit. :)

Reply:
jeeez.. ididn't know they were THAT expensive. I will make sure how to avoid using them

Reply:
The test suite in Mix is getting close to 2 minutes. It is precisely because some tests require setting up a new project. Those in particular set up a new project and a whole new VM too. It is death by a 10000 cuts. :)